upload.go

```diff
@@ -250,6 +254,10 @@ func (args requestArgs) EncodeQuery() string {
 	qs.SetOptionalInt("uploadId", args.uploadID)
 	qs.SetOptionalBool("done", args.done)
 
+	if args.associatedIndexID != nil {
+		qs.SetInt("associatedIndexID", *args.associatedIndexID)
```

Comment:
Just for consistency, do we have other lowerCamelCasedQueryParams that end in ID? If we do, is it `ID` or `Id`?

Reply:
https://github.com/sourcegraph/codeintelutils/blob/03d694caeb86d1399d2faab139a234b01c095956/upload.go#L254

Looks like `Id` is used just above.

Reply:
woorps completely misread this comment, thought it was confirming ID was right lol

Reply:
👅 I figured.

"thumbs up won't fix"